### PR TITLE
doc: add missing information about batched callbacks

### PR DESCRIPTION
**Pull request by arkadiuszos4chain**
## Description of Changes

Provide a brief description of the changes you've made.

## Linked Issues / Tickets

Reference any related issues or tickets, e.g. "Closes #123".

## Testing Procedure

Describe the tests you've added or any testing steps you've taken.

- [ ] I have added new unit tests
- [ ] All tests pass locally
- [ ] I have tested manually in my local environment

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have updated `CHANGELOG.md` with my changes

### CODE REVIEW

**Review by kuba-4chain — APPROVED** [review]

File: doc/api.md [hunk 1]
```diff
@@ -704,7 +710,7 @@ This endpoint is used to send a raw transaction to a miner for inclusion in the
 |X-SkipTxValidation|header|boolean|false|Whether we should skip overall tx validation or not.|
 |X-CumulativeFeeValidation|header|boolean|false|Whether we should perform cumulative fee validation for fee consolidation txs or not.|
 |X-CallbackToken|header|string|false|Access token for notification callback endpoint. It will be used as a Authorization header for the http callback|
-|X-CallbackBatch|header|boolean|false|Send callbacks in batches.|
+|X-CallbackBatch|header|boolean|false|Callback will be send in batch|
```
Comment (boecklim) [inline doc/api.md:713]: Callback will be **sent** in a batch

File: pkg/api/arc.yaml [hunk 2]
```diff
@@ -474,6 +474,46 @@ components:
...
+        count:
+          type: integer
+          description: number of callbacks in response
+        callbacks:
+          type: array
+          nullable: true
+          items:
+            $ref: '#/components/schemas/Callback'
+      example:
+        - batched:
```
Comment (boecklim) [inline pkg/api/arc.yaml:493]: What is this `batched` property. I appears in `arc.json` line 734 as `"batched": null,`
Can it be removed?

Comment (arkadiuszos4chain) [reply pkg/api/arc.yaml]: I based it on examples from the Callback section. I fixed it

File: doc/api.md (hunk 1 above)
Comment (arkadiuszos4chain) [reply doc/api.md]: fixed

**Review by boecklim — APPROVED** [review]

**Pull request merged**